pkgs/development/python-modules/pyfxa/default.nix

```diff
@@ -38,13 +40,15 @@ buildPythonPackage rec {
     requests
     setuptools # imports pkg_resources
     six
+    hatchling
```

Comment:
`hatchling` should go into `build-system`.

https://github.com/mozilla/PyFxA/blob/main/pyproject.toml#L2